CONTRIBUTING.rst

```diff
@@ -0,0 +1,78 @@
+Contributing
+============
+
+#. **Please sign one of the contributor license agreements below.**
+#. Fork the repo, develop and test your code changes, add docs.
+#. Make sure that your commit messages clearly describe the changes.
+#. Send a pull request.
+
+Here are some guidelines for hacking on ``google-auth-library-python``.
+
+Making changes
+--------------
+
+A few notes on making changes to ``google-auth-libary-python``.
+
+- If you've added a new feature or modified an existing feature, be sure to
+  add or update any applicable documentation in docstrings and in the
+  documentation (in ``docs/``). You can re-generate the reference documentation
+  using ``tox -e docgen``.
+
+- The change must work fully on the following CPython versions: 2.7,
+  3.4, and 3.5 across macOS, Linux, and Windows.
+
+- The codebase *must* have 100% test statement coverage after each commit.
+  You can test coverage via ``tox -e cover``.
+
+Testing changes
+---------------
+
+To test your changes, run unit tests with ``tox``::
+
+    $ tox -e py27
+    $ tox -e py34
+    $ tox -e py35
+
+Coding Style
+------------
+
+This library is PEP8 & Pylint compliant. Our pylint config is defined at
+``pylintrc`` for package code and ``pylintrc.tests`` for test code. Use
```

Comment:
Nice to know, if you want to submit your script over to handle that that's cool with me. 👍